sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -301,11 +297,10 @@ def get_copy_authorization(self, **kwargs):
     @distributed_trace
     def begin_copy_model_to(
         self,
-        model_id,  # type: str
-        target,  # type: Dict
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> DocumentModelAdministrationLROPoller[DocumentModel]
+        model_id: str,
+        target: dict,
```

Comment:
I think we should capture this as Dict[str, str] since that's what `get_copy_authorization` returns